frontend/dockerfile/linter/ruleset.go

```diff
@@ -163,5 +163,6 @@ var (
 		Format: func(cmd, file string) string {
 			return fmt.Sprintf("Attempting to %s file %q that is excluded by .dockerignore", cmd, file)
 		},
+		Experimental: true,
```

Comment:
I think we should handle this for docs generation and generate an experimental callout for such case (cc @dvdksn)

Reply:
Agreed. Maybe add a note at the top of the document

```
---
title: CopyIgnoredFile
description: Attempting to Copy file that is excluded by .dockerignore
aliases:
  - /go/dockerfile/rule/copy-ignored-file/
---

> **Note**
>
> This check is experimental.
```

Reply:
I guess a new section about how to enable experimental check would be good in https://docs.docker.com/build/checks/ and put a link to it here?

Reply:
ah, I didn't realize there is a flag to toggle them. Yes in that case a link would be good. Let's use `https://docs.docker.com/go/build-checks-experimental/` - I will create the redirect alias and the section.

Reply:
Updated to add the note for experimental checks!